rootfs/opt/qserv/gcp/create-gke-cluster.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+set -x
+
+DIR=$(cd "$(dirname "$0")"; pwd -P)
+. "$DIR/env.sh"
+
+
+# Creates a 31 nodes GKE cluster
+
+#gcloud auth login
+gcloud config set project "$PROJECT"
+SIZE=1
```

Comment:
Your comment says "31 nodes" but the SIZE variable is set to 1.

Reply:
Yes because this is default-pool size, so it will be deleted and then replace with pool-czar and pool-worker, it's also a way to reduce costs at cluster creation, and set cluster size to 31 only when needed.